e2e-tests/ignore-labels-annotations/run

```diff
@@ -0,0 +1,182 @@
+        {
+            "op": "add", 
+            "path": "/metadata/labels", 
+            "value": {
+                "notIgnoredLabel": "true",
+                "ignoredLabel": "true"
+            }
+        },
+        {
+            "op": "add", 
+            "path": "/metadata/annotations", 
+            "value": {
+                "notIgnoredAnnotation": "true",
+                "ignoredAnnotation": "true"
+            }
+        }]'
+
+	sleep 5 # waiting for reconcile
+	desc "check if annotations/labels are not deleted from service $svc_name"
+	compare_kubectl "service/$svc_name" "-manual"
+
+	desc "adding ignoreAnnotations and ignoreLabels"
+	kubectl_bin patch psmdb "$cluster" --type=json --patch '[
+        {
+            "op": "add",
+            "path": "/spec/ignoreAnnotations",
+            "value": ["ignoredAnnotation"]
+        },
+        {
+            "op": "add",
+            "path": "/spec/ignoreLabels",
+            "value": ["ignoredLabel"]
+        }
+    ]'
+	sleep 5
+	desc "check if annotations/labels are not deleted from service $svc_name"
+	compare_kubectl "service/$svc_name" "-manual"
+
+	desc "adding serviceLabels and serviceAnnotations to $expose_path"
+	kubectl_bin patch psmdb ${cluster} --type=json --patch '[
+        {
+            "op": "replace", 
+            "path": "'$expose_path'/serviceLabels", 
+			"value": {
+				"crLabel": "true",
+			}
+        },
+        {
+            "op": "replace", 
+            "path": "'$expose_path'/serviceAnnotations", 
+			"value": {
+				"crAnnotation": "true",
+			}
+        }]'
+	sleep 5
+	desc "check if not ignored labels and annotations are deleted from service $svc_name"
+	compare_kubectl "service/$svc_name" "-crmeta"
+
+	desc "removing serviceLabels and serviceAnnotations from $expose_path"
+	kubectl_bin patch psmdb ${cluster} --type=json --patch '[
+        {
+            "op": "remove", 
+            "path": "'$expose_path'/serviceLabels"
+        },
+        {
+            "op": "remove", 
+            "path": "'$expose_path'/serviceAnnotations"
+        }]'
+	sleep 5
+	desc "check if labels and annotations are not deleted from service $svc_name"
+	compare_kubectl "service/$svc_name" "-crmeta"
+
+	desc "adding other serviceLabels and serviceAnnotations to $expose_path"
+	kubectl_bin patch psmdb ${cluster} --type=json --patch '[
+        {
+            "op": "replace", 
+            "path": "'$expose_path'/serviceLabels", 
+			"value": {
+				"otherCrLabel": "true",
+				"secondCrLabel": "true",
+			}
+        },
+		{
+            "op": "replace", 
+            "path": "'$expose_path'/serviceAnnotations", 
+			"value": {
+				"otherCrAnnotation": "true",
+				"secondCrAnnotation": "true",
+			}
+		}]'
+	sleep 5
+	desc "check if not previous service labels and annotations is deleted from $svc_name"
+	compare_kubectl "service/$svc_name" "-two-service-labels-annotations"
+
+	desc "adding removing one serviceLabels from $expose_path"
```

Comment:
Does this desc reflect what is happening? I can see only one service label and service annotation being replaced.